src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -56,6 +54,23 @@ void createStation() {
      * Then 2개의 지하철역을 응답 받는다
      */
     // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다.")
+    @Test
+    void getStations(){
+
+        //given
+        List<String> requestStationNames = new ArrayList<>(Arrays.asList(new String[]{"왕십리역", "마장역"}));
+
+        requestStationNames.stream().map(StationAcceptanceTest::createStation)
+                .forEach(response -> assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value()));
+
+        // when
+        List<String> stationNames = StationAcceptanceTest.getStationNames();
+
+        //then
+        assertThat(stationNames.size()).isEqualTo(2);
+        requestStationNames.stream().forEach(stationName -> assertThat(stationNames).containsAnyOf(stationName));
```

Comment:
```suggestion
        assertThat(stationNames).containsExactly("왕십리역", "마장역");
```

containsExactly를 사용하는 건 어떨까요?

Reply:
아하 다음번에 확인해보고 사용해볼게요!